### PR TITLE
Fix link in api-conventions.md

### PR DESCRIPTION
**Pull request by it-praktyk**

### CODE REVIEW

**Comment by it-praktyk** [comment]
I just signed the CNCF Individual Contributor License Agreement so please rerun checks again.

**Comment by mrbobbytables** [comment]
Thanks for catching that @it-praktyk :) The bot took care of fixing the cncf-cla check label. 👍 


/ok-to-test

File: contributors/devel/sig-architecture/api-conventions.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Updated: 3/7/2017
 
 *This document is oriented at users who want a deeper understanding of the
 Kubernetes API structure, and developers wanting to extend the Kubernetes API.
-An introduction to using resources with kubectl can be found in [the object management overview](https://kubernetes.io/docs/tutorials/object-management-kubectl/object-management/).*
+An introduction to using resources with kubectl can be found in [the object management overview](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/declarative-config/).*
```
Comment (nikhita) [inline contributors/devel/sig-architecture/api-conventions.md]: nit: can we replace "object management overview" with "declarative object management overview" or something like that?

A page for imperative object management exists as well (https://kubernetes.io/docs/tasks/manage-kubernetes-objects/imperative-config/), so I think it'd be helpful to be precise here. :)

Comment (it-praktyk) [reply contributors/devel/sig-architecture/api-conventions.md]: Done

File: contributors/devel/sig-architecture/api-conventions.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ Updated: 3/7/2017
 
 *This document is oriented at users who want a deeper understanding of the
 Kubernetes API structure, and developers wanting to extend the Kubernetes API.
-An introduction to using resources with kubectl can be found in [the object management overview](https://kubernetes.io/docs/tutorials/object-management-kubectl/object-management/).*
+An introduction to using resources with kubectl can be found in [the declarative object management overview](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/declarative-config/).*
```
Comment (cblecker) [inline contributors/devel/sig-architecture/api-conventions.md]: Sorry for the back and forth, but I think this should actually be https://kubernetes.io/docs/concepts/overview/working-with-objects/object-management/

This is the current version of the document that https://v1-10.docs.kubernetes.io/docs/tutorials/object-management-kubectl/object-management/ goes to.

```suggestion
An introduction to using resources with kubectl can be found in [the object management overview](https://kubernetes.io/docs/concepts/overview/working-with-objects/object-management/).*
```

**Comment by cblecker** [comment]
/lgtm
/approve

Thanks for this @it-praktyk !

**Pull request merged**